### ISSUE TITLE
Fails to compile on windows?

### ISSUE THREAD

**Issue by Ayashiihito** [justjanne/powerline-go#265]
Windows 10 20H2, go version: 1.15.6 windows/amd64

```
> go get -u github.com/justjanne/powerline-go
# github.com/justjanne/powerline-go
C:\Users\username\go\src\github.com\justjanne\powerline-go\segment-exitcode.go:38:17: undefined: syscall.SIGUSR1
C:\Users\username\go\src\github.com\justjanne\powerline-go\segment-exitcode.go:42:17: undefined: syscall.SIGUSR2
C:\Users\username\go\src\github.com\justjanne\powerline-go\segment-exitcode.go:50:17: undefined: syscall.SIGSTKFLT
C:\Users\username\go\src\github.com\justjanne\powerline-go\segment-exitcode.go:52:17: undefined: syscall.SIGCHLD
C:\Users\username\go\src\github.com\justjanne\powerline-go\segment-exitcode.go:54:17: undefined: syscall.SIGCONT
C:\Users\username\go\src\github.com\justjanne\powerline-go\segment-exitcode.go:56:17: undefined: syscall.SIGSTOP
C:\Users\username\go\src\github.com\justjanne\powerline-go\segment-exitcode.go:58:17: undefined: syscall.SIGTSTP
C:\Users\username\go\src\github.com\justjanne\powerline-go\segment-exitcode.go:60:17: undefined: syscall.SIGTTIN
C:\Users\username\go\src\github.com\justjanne\powerline-go\segment-exitcode.go:62:17: undefined: syscall.SIGTTOU
C:\Users\username\go\src\github.com\justjanne\powerline-go\segment-exitcode.go:64:17: undefined: syscall.SIGURG
C:\Users\username\go\src\github.com\justjanne\powerline-go\segment-exitcode.go:64:17: too many errors
```

**Comment by dswij** [comment]
Can verify, unable to build on Windows 10 Build 19042 with Go 1.15.2

**Comment by dswij** [comment]
I believe this is because the `syscall` has some implementations that are specific to unix.

**Comment by dswij** [comment]
Actually, after looking at it for awhile, I found that the change in commit https://github.com/justjanne/powerline-go/commit/0f611fe554ebe1388ac766584f622dc5ee50bf46 might have caused this, because it used `syscall` package.

**Comment by justjanne** [comment]
Thanks for the info, I'll reconsider the change later today (I'm not entirely sure the exitcode segment even makes sense on Windows anyway).

**Comment by dswij** [comment]
Seems the most recent travis test job is failing due to similar error. Not sure why the older ones passed.

**Comment by Ayashiihito** [comment]
> Actually, after looking at it for awhile, I found that the change in commit [0f611fe](https://github.com/justjanne/powerline-go/commit/0f611fe554ebe1388ac766584f622dc5ee50bf46) might have caused this, because it used `syscall` package. Might want to reconsider the changes for this one @justjanne

Reverted this commit and yes, it was the problem, everything seems to work fine

**Comment by justjanne** [comment]
This should have been fixed by b9311ce. If the issue still occurs, please re-open the ticket
